pull_request_template.md

```diff
@@ -5,6 +5,7 @@ Include the following in PR#1:
 - [ ] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
 - [ ] Specification PDF in the form of wombat-spec-x.y.pdf
 - [ ] Specification HTML in the form of wombat-spec-x.y.html
+- [ ] If the specification produces XML Schema(s), link(s) to the [publish schema PR](https://eclipse-ee4j.github.io/jakartaee-platform/publish-xml-schemas)(s)
```

Comment:
Would you prefer to see it here in the checklist or in the _index.md file that's published later?
The schema, like the javadocs, is logically *part of* the spec, so I think it would be better to
list it on the _index.md page.

Reply:
I agree!

Reply:
A suggestion here: https://github.com/jakartaee/specification-committee/pull/30